MetaMask/metamask-extension
# Fix #23095 - Fix long account name ellipsizing

## **Description**

Fixes long account name ellipsizing in the header's AccountPicker.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/23096?quickstart=1)

## **Related issues**

Fixes: #23095 23095

## **Manual testing steps**

1. Create a long account name
2. See it ellipsized 

## **Screenshots/Recordings**



### **Before**
![Screenshot 2024-02-21 at 16 06 43](https://github.com/MetaMask/metamask-extension/assets/46655/06e49f01-5b66-4da4-9878-c8cc678beae8)



### **After**

<img width="725" alt="SCR-20240221-kvri" src="https://github.com/MetaMask/metamask-extension/assets/46655/9c42a0f2-ba7d-40af-b588-2c4bf3b04d55">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
